### PR TITLE
Show count of not filtered rows

### PR DESCRIPTION
**Pull request by Falseee**
Shows count of not filtered rows (requested by jy95 on issue #3075).

The json files must be updated, because for serverside pagination there is a new value called `totalNotFiltered`. 

Jsfiddle: http://jsfiddle.net/e3wnq1hf/3/

### CODE REVIEW

**Comment by Falseee** [comment]
@wenzhixin Should we add an option to show/hide this information ?

**Comment by djhvscf** [comment]
I think this should be optional

**Comment by Falseee** [comment]
@djhvscf Added an table option(inclusive documentation) to enable this feature.

**Comment by djhvscf** [comment]
Any update @Falseee

**Comment by Falseee** [comment]
@djhvscf  I already implemented the option, did I miss something?

**Comment by wenzhixin** [comment]
Testing the PR

File: site/docs/api/table-options.md [hunk 1]
```diff
@@ -496,6 +496,20 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
...
+
+- **Type:** `Boolean`
+
+- **Detail:**
+
+  Set `true` to show a extended version of pagination (including the count of all rows with out filters).
+
+- **Default:** `false`
+
+- **Example:** [Only Info Pagination](https://examples.bootstrap-table.com/#options/show-extended-pagination.html)
```
Comment (wenzhixin) [inline site/docs/api/table-options.md]: Example error.

We need to add `totalNotFilteredField` and `totalNotFiltered` option to docs.

File: src/bootstrap-table.js [hunk 2]
```diff
@@ -502,7 +505,11 @@
     formatRecordsPerPage (pageNumber) {
       return `${pageNumber} rows per page`
     },
-    formatShowingRows (pageFrom, pageTo, totalRows) {
+    formatShowingRows (pageFrom, pageTo, totalRows, totalNotFiltered) {
+      if(totalNotFiltered !== undefined && totalNotFiltered > 0 && totalNotFiltered < totalRows) {
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: `eslint` problem, need to use `yarn lint --fix`

File: src/locale/bootstrap-table-de-DE.js [hunk 3]
```diff
@@ -10,7 +10,12 @@
     formatRecordsPerPage (pageNumber) {
       return `${pageNumber} Zeilen pro Seite.`
     },
-    formatShowingRows (pageFrom, pageTo, totalRows) {
+    formatShowingRows (pageFrom, pageTo, totalRows, totalNotFiltered) {
+      let text =
```
Comment (wenzhixin) [inline src/locale/bootstrap-table-de-DE.js]: ?

File: site/docs/api/table-options.md (hunk 1 above)
Comment (Falseee) [reply site/docs/api/table-options.md]: Why totalNotFiltered ?
Its just a variable which contains the amount of , it makes no sense to change that (in my opinion).

totalNotFilteredField done.

File: src/bootstrap-table.js (hunk 2 above)
Comment (Falseee) [reply src/bootstrap-table.js]: done

File: src/locale/bootstrap-table-de-DE.js (hunk 3 above)
Comment (Falseee) [reply src/locale/bootstrap-table-de-DE.js]: done

**Pull request merged**